src/value.zig

```diff
@@ -2386,6 +2414,74 @@ pub const Value = extern union {
         };
     }
 
+    pub fn isVectorlike(val: Value) bool {
+        return switch (val.tag()) {
+            .aggregate, .repeated => true,
+
+            // Hack to auto-vectorize without needing to allocate a new `.repeated`
+            .zero,
+            .one,
+            .bool_true,
+            .bool_false,
+            .int_i64,
+            => true,
+
+            else => false,
+        };
+    }
+
+    /// Returns vectorlike length if values are compatible vectorlikes.
+    pub fn compatibleVectorlikes(values: anytype) ?usize {
```

Comment:
Based on usage of this function it looks like the signature should instead be:

```suggestion
    pub fn compatibleVectorlikes(a: Value, b: Value) ?usize {
```

I see one usage where only 1 parameter is passed and it doesn't make sense to me to use this function at all in that case.

And then here this function signature reveals the same problems with `isVectorLike`; this is unsound without a `Type`.